leetcode/src/236.c

```diff
@@ -0,0 +1,80 @@
+/**
+ * Definition for a binary tree node.
+ * struct TreeNode {
+ *     int val;
+ *     struct TreeNode *left;
+ *     struct TreeNode *right;
+ * };
+ */
+
+struct List {
+    struct TreeNode* node;
+    struct List* next;
+};
+
+bool findPathOfTarget(struct TreeNode* node, struct TreeNode* target, struct List* path){

```

Comment:
```suggestion
bool findTargetPath(struct TreeNode* node, struct TreeNode* target, struct List* path){
```

Avoid prepositions when you can 😄 they almost always add no real value when reading code.